src/very_basic_operations.py

```diff
@@ -2,5 +2,17 @@ def add(a, b):
     """Basic function which adds two numbers"""
     return a + b 
 
+def factorial(number):
+    n = 1
+    if type(number) != int or number < 0:
+        raise ValueError 
```

Comment:
You should do `raise ValueError("some text here")` to tell why exception was raised. This way you can raise `ValueError` with one text if number is less than 0, and another if number is not integral. And you can check text of exception in tests with `self.assertRaises(ValueError, msg="some text here")